cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -485,6 +488,20 @@ def create(self, catalog_item_class, *args, **kwargs):
         """
         cat_item = self.instantiate(catalog_item_class, *args, **kwargs)
         view = navigate_to(cat_item, 'Add')
+        if field_entry_point:
+            if provider_type == 'cloud':
+                view.basic_info.field_entry_point.fill("")
```

Comment:
Required:
`field entry_point` is common for both `infra` and `cloud` so you can put this about conditions.